src/components/Inputs/Technoeconomic/TechnoeconomicInputs.tsx

```diff
@@ -27,9 +27,8 @@ const determineScaledCapitalCost = (electricalCapacity: number) => {
   if (electricalCapacity === defaultElectricCapacity) {
```

Comment:
instead of checking electricalCapacity, how about checking if capitalCost === defaultCapitalCost? If they are not equal, which means the capital cost has been specified by users, and we will not do anything with it; otherwise scale capital cost using electric capacity.

Reply:
ignore the comment above, that is just another way I came up with to check if the capital cost is specified by users. I didn't notice your new changes when making the comment.